apps/src/lab2/views/LabViewsRenderer.tsx

```diff
@@ -38,14 +39,28 @@ const LabViewsRenderer: React.FunctionComponent = () => {
 
   const isViewingExemplar = getAppOptionsViewingExemplar();
 
+  const capitalizedLessonBackground = useAppSelector(
+    state =>
+      capitalizeFirstLetter(
+        getCurrentLesson(state)?.background || 'light'
+      ) as Theme
+  );
+
   // Set the theme for the current app.
-  const {setTheme} = useContext(ThemeContext);
+  const {setTheme} = useTheme();
   useEffect(() => {
     if (currentAppName) {
-      const theme = lab2EntryPoints[currentAppName]?.theme || DEFAULT_THEME;
-      setTheme(theme);
+      const supportedThemes = lab2EntryPoints[currentAppName]?.themes || [
+        'Dark',
+      ];
```

Comment:
nit: do we need this fallback since all labs should have a themes array defined?

Reply:
Good call; removed.